c10/cuda/CUDAStream.cpp

```diff
@@ -33,7 +33,6 @@ struct LeakyStreamInternals {
   }
 
   DeviceIndex device_index = -1;
-  int32_t stream_id = -1;
```

Comment:
This was unused. Also, it was an `int32_t` whereas StreamId is an alias for `int64_t`, weird...